package/harvester-os/files/usr/sbin/harv-install

```diff
@@ -182,27 +184,54 @@ preload_rke2_images()
 
       $rke2_tmp/bin/rke2 server &> /rke2.log &
 
+      until [ -f "/var/lib/rancher/rke2/bin/containerd" ] && [ -f "/var/lib/rancher/rke2/agent/etc/containerd/config.toml" ]
+      do
+        sleep 1
+      done
+
+      echo "Stop rke2 and remove tmporary rke2 files..."
+      pkill rke2
+      rm /rke2.log
+      rm /etc/rancher/rke2/rke2.yaml
+      rm -rf /var/lib/rancher/rke2/server
+      rm -rf /var/lib/rancher/rke2/agent/pod-manifests/*
+EOF
+
+    echo "Copying remaining images to the target location..."
+    rsync -ah --progress ${ISOMNT}/bundle/harvester/images/ $TARGET/$RKE2_IMAGES_DIR
+
+    echo "Loading images. This may take a few minutes..."
+    chroot . /bin/bash <<"EOF"
+
       export PATH=/var/lib/rancher/rke2/bin:$PATH
       export CONTAINERD_ADDRESS=/run/k3s/containerd/containerd.sock
 
+      echo "Start containerd..."
+      containerd -c /var/lib/rancher/rke2/agent/etc/containerd/config.toml -a $CONTAINERD_ADDRESS --state /run/k3s/containerd --root /var/lib/rancher/rke2/agent/containerd &> /containerd.log &
+
       # wait for containerd to be ready
       until ctr --connect-timeout 1s version&>/dev/null
       do
         sleep 1
       done
 
+      # load images
+      for i in /var/lib/rancher/rke2/agent/images/*.tar.zst; do
```

Comment:
Can you put this path prefix to a variable?

Reply:
Updated, thanks